src/lib/components/Grid/Grid.jsx

```diff
@@ -54,13 +54,13 @@ Grid.defaultProps = {
   alignItems: undefined,
   autoFlow: undefined,
   children: null,
-  columnGap: undefined,
-  columns: undefined,
+  columnGap: 'var(--rui-spacing-4)',
```

Comment:
I filed #402 to suggest limiting this kind of props only to allowed [spacing values](https://react-ui.io/foundation/spacing).